blog/_posts/2021-07-21-scala-3.0.2RC1-is-here.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: blog-detail
+post-type: blog
+by: Michał Pałka, VirtusLab
+title: Scala 3.0.1 and 3.0.2-RC1 are here!
+---
+
+Greetings from the Scala 3 team! We are glad to announce that Scala 3.0.1 and 3.0.2-RC1 are now officially out.
+
+As no critical bugs have been found in the previously released Scala 3.0.1-RC2, it has been promoted to 3.0.1, which is the first stable release after 3.0.0. Scala 3.0.2-RC1, in turn, incorporates new language improvements and bug fixes. You can expect the release of stable 3.0.2 and a release candidate for a the next version in 6 weeks from now (1st August).
+
+Keep on reading to find out what's new in 3.0.2-RC1.
```

Comment:
I suggest starting with a sentence explaining how to find the release notes of the 3.0.0 release (with a link), before this sentence.